templates/cinderbackup/config/kolla-config.json

```diff
@@ -1,5 +1,9 @@
 {
+  {{if .ServiceDebug}}
```

Comment:
Maybe instead of a conditional just have a "ServiceCommand" var that if set will override the default?